### PR TITLE
[zh-cn]sync manage-gpus/scheduling-gpus.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 1]
```diff
@@ -13,16 +13,19 @@ description: Configure and schedule GPUs for use as a resource by nodes in a clu
...
-(graphical processing units) across several nodes.
+Kubernetes includes **stable** support for managing AMD and NVIDIA GPUs
+(graphical processing units) across different nodes in your cluster, using
+{{< glossary_tooltip text="device plugins" term_id="device-plugin" >}}.
 
 This page describes how users can consume GPUs, and outlines
 some of the limitations in the implementation.
 -->
 Kubernetes 支持对若干节点上的 GPU（图形处理单元）进行管理，目前处于**实验**状态。
+Kubernetes 支持使用 {{< glossary_tooltip text="device plugins" term_id="device-plugin" >}}
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md:27]:
```suggestion
Kubernetes 支持使用{{< glossary_tooltip text="设备插件" term_id="device-plugin" >}}来
```

**Comment by tengqm** [comment]
@my-git9 Got some time to push this forward?

**Comment by my-git9** [comment]
> Got some time to push this forward?

Sure

**Comment by my-git9** [comment]
> @my-git9 Got some time to push this forward?

Sorry for the delay in processing these two days due to physical reasons

**Review by Sea-n — COMMENTED** [review]
Please also update:

- L16: feature-state version
- L19: experimental -> stable
- L34: wording
- L47: missed one sentence

**Comment by tengqm** [comment]
/lgtm

Leaving approval to @Sea-n

**Comment by Sea-n** [comment]
/approve

Thanks @my-git9 and @tengqm 😝

**Pull request merged**